### DISCUSSION TITLE
Homework Loops 20.12.24 <3

### DISCUSSION THREAD

**Discussion by danii-07** [danii-07/Programming-I#2] [Show and tell]
I developed a Java program that generates a red heart pattern. I initially used the guidance provided by [https://www.javatpoint.com/heart-pattern-in-java](https://www.javatpoint.com/heart-pattern-in-java) to understand the core functionalities of generating a heart pattern in Java. I then customized the code and employed AI assistance to gain a deeper understanding of each step, enabling me to explain the code's logic in my own words with comments. Finally, also with the help of AI, I successfully implemented a color change, resulting in a cute red heart design. 

https://github.com/danii-07/Programming-I/blob/7865bc11a62fd95881435107922baa86a37c5876/hearts.java#L1-L43

**Comment by Tam-DHBW** [comment]
Neat! I like the idea of using arcs to generate the upper parts of the heart

**Comment by Ngoc901** [comment]
Wow, this is so cute! If I had to print out a heart, I wouldn't have thought of using the distance formula to form the shape. The red color is the cherry on top! Great work! 🚀

**Comment by petra-Nam** [comment]
Wow, this is absolutely adorable! 💖 I love how you've taken the time to dive deep into understanding and customizing the heart pattern code. The fact that you combined your own learning with AI assistance to not only explain the logic but also implement a color change shows great initiative and creativity. The red heart design is so cute and really stands out—amazing job!
